bot/cogs/coding/coding.py

```diff
@@ -142,27 +150,39 @@ async def run(self, ctx: Context, language: str, *, code: str = "") -> t.Union[N
         text = None
 
         async with ctx.typing():
-            if ctx.message.attachments:  # if file is sent instead of raw code in codeblocks
+            if (
+                ctx.message.attachments
+            ):  # if file is sent instead of raw code in codeblocks
                 file = ctx.message.attachments[0]
-                if file.size > 20000:  # check the size of file exceeding max limit
+                if (
+                    file.size > 20000
+                ):  # check the size of file exceeding max limit
                     return await ctx.send("File must be smaller than 20 kio.")
 
                 buffer = BytesIO()
                 await ctx.message.attachments[0].save(buffer)
                 text = buffer.read().decode("utf-8")
-            elif code.split(" ")[-1].startswith("link="):  # if link is sent instead of file or codeblocks
-                base_url = urllib.parse.quote_plus(code.split(" ")[-1][5:].strip("/"), safe=";/?:@&=$,><-[]")
+            elif code.split(" ")[-1].startswith(
+                "link="
+            ):  # if link is sent instead of file or codeblocks
+                base_url = urllib.parse.quote_plus(
+                    code.split(" ")[-1][5:].strip("/"), safe=";/?:@&=$,><-[]"
+                )
```

Comment:
```suggestion
            # if link is sent instead of file or codeblocks
            elif code.split(" ")[-1].startswith("link="): 
                base_url = urllib.parse.quote_plus(
                    code.split(" ")[-1][5:].strip("/"), safe=";/?:@&=$,><-[]"
                )
```